src/Pages/Order/Filter.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import DateFilter from './DateFilter';
+
+export default function Filter() {
+  return (
+    <FilterContainer>
+      <div>
+        <Select>
+          <option value>Select..</option>
+          <option value="ORDER_DETAIL_NO">주문상세번호</option>
+          <option value="ORDER_NO">주문번호</option>
+          <option value="ORDRR_NAME">주문자명</option>
+          <option value="ORDRR_HP">핸드폰번호</option>
+          <option value="PRODUCT_NAME">상품명</option>
+        </Select>
+        <SearchInput type="text" placeholder="검색어를 입력하세요." />
+      </div>
+      <div>
+        <span>결제완료일 :</span>
+        <DateBtn type="button" value="전체" />
+        <DateBtn type="button" value="오늘" />
+        <DateBtn type="button" value="3일" />
+        <DateBtn type="button" value="1주일" />
+        <DateBtn type="button" value="1개월" />
+        <DateBtn type="button" value="3개월" />
+        {/* <DateFilter />
+        <DateFilter /> */}
+      </div>
+
+      <div>
+        <Button search>검색</Button>
+        <Button>초기화</Button>
+      </div>
+    </FilterContainer>
+  );
+}
+
+const FilterContainer = styled.div`
+  border: 3px solid #eee;
+  padding-left: 5px;
+  padding-bottom: 3px;
+  margin-top: 15px;
+  margin-bottom: 20px;
+  div {
+    ${({ theme }) => theme.flex('', 'center', 'row')}
```

Comment:
```suggestion
    ${({ theme }) => theme.flex('', 'center')}
```
flex-direction의 기본값은 row로 알고 있는데 맞나요~?